mig-api/action_endpoints.go

```diff
@@ -46,7 +46,7 @@ func createAction(respWriter http.ResponseWriter, request *http.Request) {
 	if err != nil {
 		panic(err)
 	}
-	ctx.Channels.Log <- mig.Log{OpID: opid, Desc: fmt.Sprintf("Received action for creation '%s'", action)}.Debug()
+	ctx.Channels.Log <- mig.Log{OpID: opid, Desc: fmt.Sprintf("Received action for creation '%v'", action)}.Debug()
```

Comment:
This is a change I'd like to be particularly mindful of.  It looks like we're "just" logging here, so I didn't worry much about this change. The `%v` format here results in the traditional behaviour such that:

```go
package main

import (
	"fmt"
)

type Thing struct {
	Id int
	Name string
}

func main() {
	fmt.Printf("Thing %v", Thing { 0, "name" })
}
```

produces:

> Thing {0 name}

Whereas replacing `%v` with `%s` produces:

> Thing {%!s(int=0) name}